src/partials/head/head-scripts.hbs

```diff
@@ -87,5 +87,17 @@ window.settings_timer=setTimeout(function () {_vwo_code.finish() },settings_tole
 {{/if}}
 
 {{#if (eq (site-profile) 'review' )}}
-<script async src="https://54.80.84.2:5000/embed.js"></script>
+<script type="application/json" class="js-hypothesis-config">
+  {
+    "openSidebar": true,
+    "openLoginForm": true
+  }
+</script>
+
+<script type="application/json" class="js-hypothesis-config">
+  {
+    "sidebarAppUrl": "https://dev-docs-review-h.kdev.msap.io/app.html"
+  }
+</script>
```

Comment:
why have 2 different script blocks? Why not just:

``` javascript
<script type="application/json" class="js-hypothesis-config">
  {
    "openSidebar": true,
    "openLoginForm": true,
    "sidebarAppUrl": "https://dev-docs-review-h.kdev.msap.io/app.html"
  }
</script>
```

Reply:
It was provided by Mehdi I haven’t tested with only one block

Reply:
Let me test first with one block.

Reply:
Testing with one block does not work, h apps doesn’t open at all